scripts/dev

```diff
@@ -120,17 +120,27 @@ populate_v2() {
     bash -c "npx hardhat run scripts/dev/populate.ts --no-compile --network dev2"
 }
 
-start() {
+up() {
   echo "Starting all services..."
   docker-compose up -d
   docker-compose logs -f
 }
 
-stop() {
+down() {
   echo "Stopping all services..."
   docker-compose down
 }
 
+start() {
+  pnpm concurrently \
+    --names "common,data-layer,builder,explorer,manager" \
+    "cd packages/commong && pnpm rsbuild dev" \
+    "cd packages/commong && pnpm rsbuild dev" \
```

Comment:
is this a typo?

Reply:
yes it was a type, thank you :) 
I pushed the fix